simulation_ws/src/deepracer_simulation/CMakeLists.txt

```diff
@@ -24,24 +15,25 @@ install(PROGRAMS
     scripts/run_local_rl_agent.sh
     scripts/run_rollout_rl_agent.sh
     scripts/run_evaluation_rl_agent.sh
-    DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+    DESTINATION lib/${PROJECT_NAME}
 )
 
-install(DIRECTORY launch/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
-)
-install(DIRECTORY config/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/config
-)
-install(DIRECTORY meshes/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/meshes
-)
-install(DIRECTORY models/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/models
-)
-install(DIRECTORY urdf/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/urdf
-)
-install(DIRECTORY worlds/
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/worlds
+install(DIRECTORY launch config models worlds
```

Comment:
How come you don't install meshes, urdf?

Reply:
We have moved meshes to models folder. 
Converted urdf to sdf and moved it to models folder.

Reply:
Any reason for moving them? In general we'd like to change only the things that are needed for ROS2. If there is a reason (e.g. if in ROS2 the convention around directory structure has changed) that's fine, but right now it just seems like there are a bunch of changes that are not strictly needed. This makes reviewing more difficult.

Reply:
Yes. They are moved as per ROS2 convention. So there are structure changes.
We took reference from following link:
https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo/models/turtlebot3_burger.

Updated structure is as follows:

/models
  - /easy_track
    - meshes
    	- scripts
    	- textures
    - model.config
    - model.sdf
  - /hard_track
    - dae/png/jpg files
    - model.config
    - model.sdf
  - /medium_track
    - meshes
    	- scripts
    	- textures
    - model.config
    - model.sdf
  - /racer_car
    - meshes
    - model.config
    - model.sdf

NOTE: Older launch files will be removed in the next PR.

Reply:
Actually in ROS1 code for hard_track, dae and images files were inside hard_track folder. There was not textures folder. I will make that change and commit it again.